### PR TITLE
Added "cover" attributes for Page blocks

### PR DESCRIPTION
**Pull request by gabrocheleau**
While the icons of Page blocks could easily be modified with the API, there was no functionality to modify the covers. I added the relevant attributes to Page Blocks and Collection View Page Blocks.

### CODE REVIEW

File: notion/block.py [hunk 1]
```diff
@@ -772,6 +777,12 @@ class CollectionViewPageBlock(CollectionViewBlock):
         api_to_python=add_signed_prefix_as_needed,
         python_to_api=remove_signed_prefix_as_needed,
     )
+    
+     cover = field_map(
```
Comment (logicalicy) [inline notion/block.py:781]: Very subtle but I think this line's indentation needs fixing

Comment (gabrocheleau) [inline notion/block.py:781]:
```suggestion
    cover = field_map(
```

Comment (gabrocheleau) [reply notion/block.py:781]: Thank you!

**Comment by jamalex** [comment]
Thanks, folks! Looks good.

**Pull request merged**